content/es/continuous-delivery.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Entrega Continua (CD)
+status: Completed
+category: Concepto
+---
+
+## ¿Qué es?
+
+La entrega continua, a menudo abreviada como CD, es un conjunto de prácticas en las que los cambios de código se implementan automáticamente en un entorno de aceptación (o, en el caso de la implementación continua, en producción). El CD incluye procedimientos cruciales para garantizar que el software se pruebe adecuadamente antes de la implementación y proporciona una forma de revertir los cambios si se considera necesario. La integración continua (CI) es el primer paso hacia la entrega continua (es decir, los cambios deben fusionarse limpiamente antes de probarse e implementarse).
+
+## Problema que aborda
+
+La implementación de actualizaciones [confiables](/reliability/) se convierte en un problema a escala. Idealmente, implementaríamos con más frecuencia para ofrecer un mejor valor a los usuarios finales. Sin embargo, hacerlo manualmente se traduce en altos costos por cada cambio. Históricamente, para evitar estos costos, las organizaciones liberaban con menos frecuencia, implementaban más cambios a la vez y aumentaban el riesgo de que algo salga mal.
```

Comment:
```suggestion
La implementación de actualizaciones de [fiabilidad](/reliability/) se convierte en un problema a escala. Idealmente, implementaríamos con más frecuencia para ofrecer un mejor valor a los usuarios finales. Sin embargo, hacerlo manualmente se traduce en altos costos por cada cambio. Históricamente, para evitar estos costos, las organizaciones liberaban con menos frecuencia, implementaban más cambios a la vez y aumentaban el riesgo de que algo salga mal.
```
Fiabilidad: https://www.servicenow.com/es/products/it-operations-management/what-is-site-reliability-engineering.html